tests/components/remote/test_init.py

```diff
@@ -114,6 +116,28 @@ def test_learn_command(self):
         assert call.service == SERVICE_LEARN_COMMAND
         assert call.data[ATTR_ENTITY_ID] == "entity_id_val"
 
+    def test_delete_command(self):
+        """Test delete_command."""
+        delete_command_calls = mock_service(
+            self.hass, remote.DOMAIN, SERVICE_DELETE_COMMAND
+        )
+
+        common.delete_command(
+            self.hass,
+            entity_id="entity_id_val",
+            device="test_device",
+            command=["test_command"],
+        )
+
+        self.hass.block_till_done()
+
+        assert len(delete_command_calls) == 1
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6356951)